panel/theme/css/fast_variables.css

```diff
@@ -1,5 +1,6 @@
 :host {
+  --background-color: var(--fill_color);
   --bokeh-base-font: Open Sans, Roboto, sans-serif;
   color: var(--neutral-foreground-rest);
-  background-color: var(--fill-color);
+  background-color: var(--background-color);
```

Comment:
This change I would say is not 100% correct. The `--fill-color` is the correct one to use for fast.

Reply:
Other wise you should define `--background-color: var(--fill-color)`?

Reply:
I did. Look above.